tests/integration/dev.bats

```diff
@@ -144,3 +144,24 @@ function teardown() {
 	runc exec -t test_exec sh -c "ls -l /proc/self/fd/0; echo 123"
 	[ "$status" -eq 0 ]
 }
+
+@test "runc run [hostname domainname]" {
+	requires root
+
+	update_config ' .process.args |= ["sh"]
+			| .hostname = "myhostname"
+			| .domainname= "mydomainname"'
+
+	runc run -d --console-socket "$CONSOLE_SOCKET" test_allow_char
```

Comment:
nit: the `test_allow_char` is probably a copy-paste leftover. You can in fact use any name.